src/store/user/slice.tsx

```diff
@@ -0,0 +1,34 @@
+import { UserState } from '../../types/user';
+import { createSlice } from '@reduxjs/toolkit';
+import { signInAsync, signUpAsync } from './actions';
+
+const initialState: UserState = {
+  token: '',
+  email: '',
+  firstName: '',
+  lastName: '',
+  id: 0,
+};
+export const userSlice = createSlice({
+  name: 'user',
+  initialState,
+  reducers: {},
+  extraReducers(builder) {
+    builder.addCase(signInAsync.fulfilled, (state, action) => {
+      const responseData = action.payload;
+      return {
+        token: responseData.accessToken,
+        ...responseData.user,
```

Comment:
Will we have password field in our redux store?

Reply:
No we won't.